api/src/main/java/org/apache/brooklyn/api/entity/Entity.java

```diff
@@ -314,4 +361,80 @@
          */
         <T> void emit(Sensor<T> sensor, T value);
     }
+    
+    @Beta
+    public interface PolicySupport {
+        /**
+         * @return an immutable thread-safe view of the policies.
+         */
+        Collection<Policy> getPolicies();
```

Comment:
Do you mean `public interface PolicySupport extends Iterable<Policy>`? I'd prefer not for a couple of reasons. First if you wrote `entity.policy().` the auto-complete would mix the addPolicy/removePolicy with the other methods on Iterable. Second, the intent is consistency for things on the Entity api - for `config()`, `sensor()` and `subscriptions()` they don't return Iterable so I'm inclined to not have policies/enrichers return an iterable either.